src/viam/sdk/services/motion/motion.hpp

```diff
@@ -87,6 +90,122 @@ struct motion_configuration {
 /// specific motion implementations. This class cannot be used on its own.
 class Motion : public Service {
    public:
+    /// @enum PlanState
+    /// @brief Describes the possible states a plan can be in.
+    /// @ingroup Motion
+    enum class PlanState : uint8_t {
+        in_progress = 0,
+        stopped = 1,
```

Comment:
Are the numeric values important, or could you leave them off? Also, enum values are constants so consider adorning them as such.

Reply:
definitely don't need the numbers! will drop.